minio/signer.py

```diff
@@ -40,7 +40,7 @@
 _SIGN_V4_ALGORITHM = 'AWS4-HMAC-SHA256'
 
 # Hardcoded S3 header value for X-Amz-Content-Sha256
-_UNSIGNED_PAYLOAD = b'UNSIGNED-PAYLOAD'
+_UNSIGNED_PAYLOAD = 'UNSIGNED-PAYLOAD'
```

Comment:
I've described the reason of declaring _UNSIGNED_PLAYLOAD as bytes in the PR description and not in the code (sorry!). This is a workaround to support both 2.7 and 3.x versions.